### PR TITLE
docs: add webinar video to readme and doc home page

### LINKED ISSUE #871: add webinar video to readme and doc home page

### PR DESCRIPTION
**Pull request by dhanusaputra**
fixes #871

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,11 @@ application's lifecycle.
 
 Read more about Kargo in our [docs](https://kargo.akuity.io) or get hands-on
 right away by following our 
-[Quickstart documentation](https://kargo.akuity.io/quickstart)!
+[Quickstart documentation](https://kargo.akuity.io/quickstart) or its webinar video!
+
+<center>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yaZc0DdeLKk?si=BACwCHp8IKAIa7Ef" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (krancour) [inline README.md:31]: Even though this works in regular Markdown, GitHub will not render it.

File: docs/docs/10-overview.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ application's lifecycle.
 
 Kargo's goal is to provide an intuitive and flexible layer "above" existing GitOps tooling, wherein you can describe the relationships between various application instances deployed to different environments as well as procedures for progressing changes from one application instance's source of truth to the next.
 
+:::info
+If you prefer learning through video, check out the video webinar of this guide!
```
Comment (krancour) [inline docs/docs/10-overview.md:21]: This text might need to change. Unlike our old video quickstart, this webinar is less of a tutorial and more of a conversational introduction to Kargo.

I will try to come up with some suggested text tomorrow, or @morey-tech might have ideas, too.

Comment (morey-tech) [reply docs/docs/10-overview.md:21]: Probably something like this:
```suggestion
Watch the launch webinar with Kelsey Hightower and Jesse Suen to see a live demo and discussion of Kargo!
```

**Review by morey-tech — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -25,7 +25,9 @@ application's lifecycle.
 
 Read more about Kargo in our [docs](https://kargo.akuity.io) or get hands-on
 right away by following our 
-[Quickstart documentation](https://kargo.akuity.io/quickstart)!
+[Quickstart documentation](https://kargo.akuity.io/quickstart) or its webinar video!
```
Comment (krancour) [inline README.md:28]:
```suggestion
[Quickstart documentation](https://kargo.akuity.io/quickstart) or watch the launch webinar with Kelsey Hightower and Jesse Suen to see a live demo and discussion of Kargo:
```

Comment (krancour) [reply README.md:28]: LGTM after this.

**Review by krancour — APPROVED** [review]

**Comment by krancour** [comment]
Thank you @dhanusaputra !

**Pull request merged**